Airports/EGSY/Positions.txt

```diff
@@ -0,0 +1,2 @@
+EGSY_TWR:St Athan Tower:122.870:SYT:F:EGSY:TWR:-:-:3646:3657:N051.23.48.000:W003.20.36.000::
```

Comment:
Real world 122.865

Cardiff Approach's allocation is 3601 - 3657, but the AIP also lists a subset of this as being 3646-3657 - I thought that since EuroScope always starts from the first squawk in the range, any local assignment would be best starting at 3646